Feature/as a service

DevHDL opened the pull request:
**기능**
as-a-Service 내용 작성 완료.

**의논점**
- png, jpg 등 첨부파일은 어떻게 저장할 것인가?
- as-a-Service의 내용은 적절한가?

DevHDL committed:
feat: Add as-a-Service

DevHDL committed:
#2 docs: as-a-Service란? 내용 추가

DevHDL committed:
#2 docs: as-a-Service 비교 추가

DevHDL committed:
docs: update as-a-Service 비교 (#2)

as-a-Service 비교 중, 인용글 문구 개선

DevHDL commented:
**png, jpg 등 첨부파일은 어떻게 저장할 것인가?**
1. assets 디렉토리 위치
2. assets 내 디렉토리 규칙

yengerche commented:
*의견*
1. assets 디렉토리 위치

> 현 위치가 적합해 보임
> 다만 이 경우 assets 디렉토리 내부에 cs, ml 등 서브 폴더가 동일이 있어야 할 것으로 예상됨

2. assets 내 디렉토리 규칙

> 우선 장기적으로는 각 파일의 형식 통일이 필요해 보임
> 다음과 같은 양식을 제안함 "/assets/img/cs/00000.png"
> 현 상황에서는 이미지를 제외하고 첨부할 가능성이 있는 것이 없음으로, 다음과 같이 제안함 "/assets/png/cs/0000.png"
> 번거롭지만 특정한 고유 식별 번호를 부여하는 방식도 좋을 것으로 생각됨
> 예시 "cs(서브 디렉토리 약자), 0001(서브 디렉토리 내의 문서 번호), 01(이미지 번호)", "cs000101.png"

DevHDL commented:
@yengerche 좋은 의견입니다. 수정하는대로 commit 올리겠습니다.

DevHDL committed:
docs: Update iamge paths and fix as-a-Service (#2,#4)

Image path 설정
- assets 디렉토리는 그대로 존치
- 다음과 같이 첨부파일 형식 통일
    ```
    이미지 파일 : assets/img/{분류}/{용어}_{번호}.png
    pdf 파일 : assets/img/{분류}/{용어}_{번호}.pdf
    ```

DevHDL commented:
- 다음과 같이 첨부파일 형식 통일
  ```
  이미지 파일 : assets/img/{분류}/{용어}_{번호}.png
  pdf 파일 : assets/img/{분류}/{용어}_{번호}.pdf
  ```

yengerche reviewed (approved):
Accept
